guides/common/assembly_preparing-for-disaster-recovery-and-recovering-from-data-loss.adoc

```diff
@@ -4,6 +4,10 @@ include::modules/con_preparing-for-disaster-recovery-and-recovering-from-data-lo
 
 include::modules/con_overview-of-recommended-disaster-recovery-plans.adoc[leveloffset=+1]
 
+include::modules/proc_retrieving-status-of-services-by-using-web-ui.adoc[leveloffset=+1]
+
+include::modules/proc_retrieving-status-of-services-by-using-cli.adoc[leveloffset=+1]
+
```

Comment:
I'd prefer to keep them in the individual disaster recovery assemblies. They serve a purpose there -- after disaster recovery (or DR testing), the module is supposed to help users verify the overall health of their post-DR environment. But perhaps just the CLI module is enough, see also my comment about possibly dropping the web UI module.